app/views/users/show.slim

```diff
@@ -71,4 +71,5 @@
       small
         = I18n.tw("home.like_to_talk")
         br
-        == t("home.send_us_an_email", mail_to: mail_to(Whitelabel[:email], "E-Mail", title: "E-Mail"), twitter_link: link_to_twitter(Whitelabel[:twitter]))
+        == contact_text
```

Comment:
looks like you already started to use a helper?

Reply:
Oh yeah, that was a left over. I started using a helper but it felt more messy with things like HTML-Escaping and spaces between translations. I've pushed a commit that re-implements the helper. Let me know what you think. Maybe we could improve it by moving the links out of the translation or something.